json/test/targeting/negative/fractional-invalid-bucketing.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
All these tests used to be embedded in `flag` json objects, which was redundant. This PR converts all tests which just test targeting validation to purely targeting tests, since we can now validate targeting alone.